### PR TITLE
LPS-126440

### PR DESCRIPTION
**Pull request by alexcandidos**

### CODE REVIEW

**Comment by Tim-Cao** [comment]
Hi @alexcandidos, this case is already covered by WebContentWithCustomstructures#AddWCWithFirstComplexStructure

**Comment by alexcandidos** [comment]
Hey @Tim-Cao , we prefer to keep this test as is in Data Engine scope. Thanks !

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path [hunk 1]
```diff
@@ -89,6 +89,21 @@
 	<td>//ul[@id='translationManager']//span[contains(.,'${key_localizedLanguage}')]</td>
 	<td></td>
 </tr>
+<tr>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path:92]: Duplicate

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path [hunk 2]
```diff
@@ -89,6 +89,21 @@
...
+	<td>TRANSLATION_LANGUAGE</td>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path:93]: Duplicate

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path [hunk 3]
```diff
@@ -89,6 +89,21 @@
...
+	<td>//div[contains(@class,'menu-open')]//a[@data-value='${key_localeUnderscore}']//span[contains(text(),'${key_locale}')]</td>
+	<td></td>
+</tr>
+<tr>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path:97]: Duplicate

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path [hunk 4]
```diff
@@ -89,6 +89,21 @@
...
+	<td>TRANSLATION_STATUS</td>
+	<td>//div[contains(@class,'open lfr-icon-menu-open')]//span[contains(text(),'${key_locale}')]//span[contains(text(),'')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>TRANSLATION_FLAG_LANGUAGE</td>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path:103]: Duplicate

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedSimpleFieldsStructure.testcase [hunk 5]
```diff
@@ -69,13 +69,37 @@ definition {
...
+			category = "Content & Data",
+			portlet = "Web Content");
+
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(
+			structureDescription = "Test",
+			structureName = "Test");
+
+		Click(locator1 = "Button#LOCALIZATION_GENERIC");
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedSimpleFieldsStructure.testcase:89]: Use existing macro

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3610
View total diff https://github.com/liferay-echo/liferay-portal/pull/3610/commits/f1d9f0929002054c07bff2fede54cc2a3f2a927a

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**